docs/developers.md

```diff
@@ -117,9 +117,9 @@ To see all available tasks run: `./gradlew tasks --all`
 `Logisim-evolution` is often updated.
 The [branch `main`](https://github.com/logisim-evolution/logisim-evolution/tree/main)
 is the place where all the work on next release happens.
-Once the code reaches the point it is ready for the next public release, it will
-be merged into the [`master` branch](https://github.com/logisim-evolution/logisim-evolution/tree/master) and released.
-But if you want to contribute, or even just see what we are currently working on, checkout the `main` branch
+Once the code reaches the point it is ready for the next public release, the public release will be created from `main`
```

Comment:
Can we somehow avoid using twice "public release"? E.g.: "the release packages will be created [...]"

Reply:
I decided to simplify the sentence.